src/experiments/run_test.py

```diff
@@ -1,42 +1,47 @@
+from pathlib import Path
 from src.utils.dptest import extract_info_from_dptest_txt, get_head_weights
 from src.infra import Record
 import subprocess
 import os
 import numpy as np
-from pathlib import Path
-ROOT_PATH = "/mnt/data_nas/public/multitask/" # may need update
 
 
-def run_single_head_dptest(run_id:str, ckpt:int, head:str):
+def run_single_head_dptest(exp_path:str, ckpt:int, head:str):
     dptest_res = {}
```

Comment:
ideally, this should be a general function that can be used for single task test.

Reply:
Sure! Maybe in another PR?